stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -92,6 +92,12 @@ public struct ${Self} {
 ${Availability(bits)}
 extension ${Self}: CustomStringConvertible {
   /// A textual representation of the value.
+  ///
+  /// For all finite values, the value of this property is a string that can be
```

Comment:
Suggest:
> For any finite value, this property provides a string ...

Reply:
Sure — changed in 28ad84b4145a1a64170b667ea84de4cc8f61e059.